packages/paste-core/components/modal/src/ModalFooterActions.tsx

```diff
@@ -10,29 +10,22 @@ export interface ModalFooterActionsProps {
   justify?: Justify;
 }
 const ModalFooterActions = React.forwardRef<HTMLDivElement, ModalFooterActionsProps>(
-  ({children, element = 'MODAL_FOOTER_ACTIONS', justify}, ref) => {
-    const count = React.Children.count(children);
-    return (
-      <Box
-        display="flex"
-        justifyContent={justify === 'start' ? 'flex-start' : 'flex-end'}
-        flexShrink={justify === 'start' ? null : 0}
-        flexWrap="wrap"
-        flexGrow={1}
-        element={element}
-        alignItems="center"
-        ref={ref}
-      >
-        {React.Children.map(children, (child, index) => {
-          return (
-            <Box marginRight={count !== index + 1 ? 'space50' : null} element={`${element}_ITEM`}>
```

Comment:
issue: unfortunately, the remove of this styling hook for a customer is a breaking change. If a customer is relying on that hook, we might break their app. We should be very conscious about whether we need to remove styling hooks from components.

Reply:
@TheSisb and I talked about this too.... This is the wrong way to space the elements inside the FooterActions. 

Is there a happy medium where we keep the iterating over children part to add the element but also use columnGap instead of marginRight?

Reply:
Oh, maybe I mispoke but let's leave Modal as is but do the right way in SideModal for now